src/codeflare_sdk/job/jobs.py

```diff
@@ -169,3 +169,6 @@ def status(self) -> str:
 
     def logs(self) -> str:
```

Comment:
Did you test the logging functionality with a notebook. I think it might break with MCAD logs due to the `None` as the second arg.